components/StoryList/index.js

```diff
@@ -16,19 +18,23 @@ function StoryList() {
   }
 
   return (
-    <StoryListContainer>
+    <>
       <Header>My Stories</Header>
-
-      {stories.map((story) => (
-        <Link key={story.id} href={`/stories/${story.id}`}>
-          <StoryListCard>
-            <StoryCoverImage src={story.coverImage} alt={story.title} />
-            <StoryTitle>{story.title}</StoryTitle>
-            <StoryCreationDate>{story.dateCreated}</StoryCreationDate>
-          </StoryListCard>
-        </Link>
-      ))}
-    </StoryListContainer>
+      <Link href="/CreateStory">
+        <AddStoryButton type="button">Write a new Story</AddStoryButton>
+      </Link>
```

Comment:
```suggestion
      <Link href="/CreateStory">
           Write a new Story
      </Link>
```

Do not nest a button in a link element as links and buttons have different purposes. [See this comparison to learn more. :)](https://css-tricks.com/a-complete-guide-to-links-and-buttons/)
-> but you can style the link so that the link look likes a button :)

Reply:
Thx for this advice and the Link! i did the changes.